test/OpenIddict.Server.Tests/OpenIddict.Server.Tests.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net461;net472;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Moq" Version="4.13.1" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.4.0" />
```

Comment:
This dependency is automagically added by Arcade and specifying it explicitly causes the exception seen in the CI logs.

Reply:
Removed. I swore I had seen another of the Test projects including it, but looking again I can't. Maybe it was because it showed up in the NuGet Package Manager window in VS? Either way, removed.